packages/utilities/psammead-styles/src/__snapshots__/global-styles.test.jsx.snap

```diff
@@ -0,0 +1,5 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
This file doesn't contain `Sans-Serif` which is passed in `it('should handle global-styles with fonts props', () => {`?